fontforge/splineutil.c

```diff
@@ -6541,7 +6529,7 @@ return;
         }
       }
       free(sf->layers); sf->layers = NULL;
-    }   
```

Comment:
I really don't like these whitespace changes. They just make blame harder to use.